src/index.md

```diff
@@ -20,6 +20,8 @@ Less is more.  Zapps are as simple as we can make them.
 - [Join the Community!](#join-the-community)
 
 
+https://www.youtube.com/watch?v=Q33LgKAwpZU
```

Comment:
Really?  There's no markup required at all?  O.o

Reply:
Haha nope, less is more right? 😄

Reply:
I'm surprised but I guess it's okay!

Mostly I'm thinking someone maintaining this might be a bit unlikely to look at the text, and realize that having the link alone, on its own line, is Magical.

Not the worst, though.  Just... my eyebrows; they escalated.